tests/console/zypper_extend.pm

```diff
@@ -46,6 +56,10 @@ sub run {
 
     #Search for a package (star
     zypper_call 'se star';
+    zypper_call 'se "sta"';
```

Comment:
I would not use `""` when they are not needed.

Reply:
> I would not use `""` when they are not needed.

Yes, it's true, but I decided to include just in one search the " " as described in the guide pdf : https://documentation.suse.com/sles/12-SP4/pdf/book-sle-admin_color_en.pdf (p.52)

Reply:
But it's not only in one search, no big deal. It's ok